Neos.Neos/Classes/Fusion/Helper/BackendHelper.php

```diff
@@ -40,6 +41,39 @@ public function interfaceLanguage(): string
         return $this->userService->getInterfaceLanguage();
     }
 
+    public function isEditMode(ActionRequest $request): bool
```

Comment:
This method could be static as far as I can tell
```suggestion
    public static function isEditMode(ActionRequest $request): bool
```